nc_setup/liquid/core/templates/layout.user.php

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
```

Comment:
Do we need to clone all this HTML here? How much different is it from the original `layout.user.php`? My concern is that the upstream version will evolve and it will be difficult to update our copy.